sound/soc/sof/mediatek/mt8186/mt8186-loader.c

```diff
@@ -24,7 +24,7 @@ void mt8186_sof_hifixdsp_boot_sequence(struct snd_sof_dev *sdev, u32 boot_addr)
 
 	/* set core boot address */
 	snd_sof_dsp_write(sdev, DSP_SECREG_BAR, ADSP_ALTVEC_C0, boot_addr);
-	snd_sof_dsp_write(sdev, DSP_SECREG_BAR, ADSP_ALTVECSEL, ADSP_ALTVECSEL_C0);
+	snd_sof_dsp_write(sdev, DSP_SECREG_BAR, ADSP_ALTVECSEL, ADSP_ALTVECSEL_C0 | ADSP_ALTVECSEL_C1);
```

Comment:
For the existing mt8186 platform, we are writing BIT(1) to dsp, but after this change, we are writing BIT(0) | BIT(1) to dsp, is this good?

Reply:
please also fix checkpatch warnings reported: https://sof-ci.01.org/linuxpr/PR4100/build2460/checkpatch/index.html

Reply:
The change in the mt8186-loader.c is compatible on both mt8186 and mt8188.
The register controls booting the DSP core with the
default address or the user specified address. Both mt8186
and mt8188 should boot with the user specified boot in the driver.
The usage of the register is the same on both SoC, but the control bit is
different on mt8186 and mt8188, which is bit 1 on mt8186 and bit 0 on mt8188.
Configure the redundant bit has no side effect on both SoCs.

Above information is also added in the commit message for future reference.